client/sidebar/RoomList/normalizeSidebarMessage.js

```diff
@@ -1,8 +1,81 @@
 import { escapeHTML } from '@rocket.chat/string-helpers';
+import { TAPi18n } from 'meteor/rocketchat:tap-i18n';
 
 import { filterMarkdown } from '../../../app/markdown/lib/markdown';
 
 export const normalizeSidebarMessage = (message, t) => {
+	if (message.t) {
+		switch (message.t) {
```

Comment:
This code already exists on the MessageType object, it shouln't be duplicated here.